scripts/test.sh

```diff
@@ -99,3 +99,12 @@ if [[ $TEST_SUITE == "unit" ]]; then
 	#     done
 	# fi
 fi
+
+# elif [[ $TEST_SUITE == "integration" ]]; then
+# 	cd $SNAP_PLUGIN_SOURCE
+# 	if [[ $SNAP_PLUGIN_SOURCE == "" ]]; then
+# 	echo "env var SNAP_PLUGIN_SOURCE not set, should be location of snap-plugin-publisher-hana"
+# 	exit 1
+# 	fi
+# 	go test -v --tags=integration ./...
+# fi
```

Comment:
I left it because it is easier for someone to try fixing the integration test without having to add that code or go to another repo to copy it. Also, it will need to be there once the test does work.. It's just commented out. But I could remove it for now if it would be better that way.